packages/prisma/migrations/20220323162642_events_hide_notes/migration.sql

```diff
@@ -0,0 +1,44 @@
+-- AlterTable
```

Comment:
@zomars There are some missing migrations that went here due being outdated since prisma change to 3.10

Reply:
Let's remove these from here and follow up in #2162